ale/drivers/messenger_drivers.py

```diff
@@ -124,8 +123,27 @@ def detector_center_sample(self):
     def detector_center_line(self):
         return float(spice.gdpool('INS{}_BORESIGHT'.format(self.ikid), 0, 3)[1])
 
+    @property
+    def sensor_model_version(self):
+        """
+        Returns
+        -------
+        : int
+          model version
+        """
+        return 2
 
-class MessengerMdisIsisLabelNaifSpiceDriver(IsisLabel, Driver, NaifSpice, Framer):
+    @property
+    def usgscsm_distortion_model(self):
+        return {
+            "transverse": {
+                "x" : self.odtx,
+                "y" : self.odty
+                }
+            }
+
```

Comment:
Copied this function from the IsisLabelNaifSpiceDriver based on the assumption that the distortion model wouldn't change depending on whether or not we're reading from an IsisLabel or a Pds3Label